doc/footer.html

```diff
@@ -17,5 +17,126 @@
 </a> $doxygenversion
 </small></address>
 <!--END !GENERATE_TREEVIEW-->
+<script type="text/javascript">
+//<![CDATA[
+function myStringCompare(str1, str2) {
```

Comment:
The reason why I used my own myStringCompare was to cover cases with numbers.

For example: 
If I do had two buttons one called OpenCV 9.1 and Opencv 10.2 it would say that OpenCV 10.2 would come first, since the ASCII code of 1 is smaller than 1.